docs/learn/metadata-stores/deltalake.md

```diff
@@ -0,0 +1,32 @@
+# DeltaMetadataStore
+
+Delta Lake is a object-store-backed metadata backend.
+It keeps one Delta table per feature, appends new metadata versions, and leans on Polars / Narwhals for provenance calculations.
+
+## Installation
+
+The backend relies on [`deltalake`](https://delta-io.github.io/delta-rs/python/)
+which ships with Metaxy’s `delta` dependency group.
+
+## Object Stores
+
+Point `root_path` at any supported URI (`s3://`, `abfss://`, `gs://`, `file://`,
+`local://`, …) and forward credentials with `storage_options`.
+The dict is passed
+verbatim to `deltalake`, so you can reuse the same keys you would use with delta-rs directly.
```

Comment:
either name it `deltalake` or `delta-rs` (I'd prefer `deltalake` since it's the Python package that's called under the hood - pretty sure Polars just does that instead of using the Rust SDK), two different names for the same thing may be confusing for users who don't know much about the ecosystem. 

Also, either a reference to the corresponding docs or a link to our own API docs would be useful: users could immediately learn more about these options by going there